gig/views.py

```diff
@@ -101,6 +102,29 @@ def form_valid(self, form):
 
         return result
 
+class CommentsView(LoginRequiredMixin, TemplateView):
+    template_name='gig/gig_comments.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        gig = Gig.objects.get(id=self.kwargs['pk'])
+        context['gig'] = gig
+        try:
+            context['comments'] = GigComment.objects.filter(gig__id=self.kwargs['pk']).order_by('created_date')
+        except GigComment.DoesNotExist:
+            context['comments'] = None
+        return context
+
+    def post(self, request, **kwargs):
+        text = request.body.decode('ascii')
```

Comment:
Would using `request.POST` let us access this information without this manual processing?

Reply:
yes!